fgd/point/comp/comp_prop_cable.fgd

```diff
@@ -0,0 +1,65 @@
+	[
+	group(target_source): "Group" : : "Each set of cables must have a group name. Each name will be compiled to one cable."
+	nextkey(target_destination) : "Next Cable" : : "Name of the next cable along this path."
+
+	slack(integer) : "Slack" : 25 : "How much extra length the cable has, for 'catenary' mode (by default it has the length between its two endpoints in the editor)."
+
+
+	// Valve's interplator keyvalue just happens to provide the right visuals we want.
+	positioninterpolator[engine](integer): "Type": 2
+	positioninterpolator(choices) : "Type" : 2 : "How to interpolate the cable. Straight makes it straight. Spline uses a spline curve, which smoothly blends between points (no visuals). Catenary makes it hang down, like the original move_rope." =
+		[
+		0: "Straight"
+		1: "Spline Curve"
+		2: "Catenary "
+		]
+
+	segments(integer) : "Segments" : 2 : "Number of nodes to generate between each cable. Higher values make smoother cables, but produce more faces."
+	sides(integer) : "Sides" : 8 : "Number of faces for each cylindrical section. Only the value at the start of the cable is used."
+	radius(float) : "Radius" : 1.0 : "Radius of this cable section."
+
+	material(material) : "Cable Material" : "models/cables/generic_black" : "The material to use for the cable. Must be a model material."
+	mat_scale(float) : "Lengthwise Material Scale" : 1 : "This changes the texture resolution along the cable, relative to the amount of width used. Larger values stretch the texture and smaller values scrunch it up."
+	mat_rotate[engine](boolean) : "Rotate Material" : 0
+	mat_rotate(choices) : "Material Orientation" : 0 : "Controls which direction the cable runs in the material, if your texture is rotated." = 
+	[
+		0 : "Vertical"
+		1 : "Horizontal"
+	]
+	u_min(float): "Width Start" : 0.0 : "The distance along the texture to start. 0 is the left/bottom side, 1 is the right/top side. This allows using only part of the texture, if it contains multiple different cable styles."
+	u_max(float): "Width End"   : 1.0 : "The distance along the texture to end. 0 is the left/bottom side, 1 is the right/top side. This allows using only part of the texture, if it contains multiple different cable styles."
+	
+	// A selection of prop_static's keyvalues, excluding some that are rather irrelevant.
+	linedivider_staticprop[!engine](string) readonly : "----------------------------------------------------------------------------------------------------------" : ""
+
+	renderamt[since_L4D](integer) : "Alpha" : 255 : "Alpha of the fade, where 0 = fully transparent and 255 = fully opaque."
+	rendercolor[since_L4D](color255) : "Color (R G B)" : "255 255 255"
+
+	screenspacefade[until_ASW](boolean) : "Screen Space Fade" : 0 : "The method by which the fading distance should be determined. If 'No', the fade distances is the distance from the player's view to the object, in inches. " +
```

Comment:
These seem to have the `[since_L4D]` and `[until_ASW]` still, intentional?

Reply:
Woops! Not intentional no. Surprised it even compiled with that. I fixup'd those 2 commits (`comp_prop_cable` and `comp_prop_rope` had this).

Every field stayed except `screenspacefade`, as what we did with `prop_static` when removing ASW from the repo. It is marked as obsolete & shoots out a warning on Chaos.